client/components/web-preview/index.jsx

```diff
@@ -70,6 +71,10 @@ const WebPreview = React.createClass( {
 		if ( this.props.previewUrl !== 'about:blank' ) {
 			this.setIframeUrl( this.props.previewUrl );
 		}
+
+		if ( this.props.showPreview ) {
+			componentClasses( document.documentElement ).add( 'no-scroll' );
```

Comment:
Can we just start using [`classList`](http://caniuse.com/#feat=classlist) instead?

```js
document.documentElement.classList.add( 'no-scroll' );
```